docs/hosting/scaling/concurrency-control.md

```diff
@@ -0,0 +1,31 @@
+---
+contentType: explanation
+---
+
+# Concurrency control
+
+In regular mode, n8n doesn't limit how many production executions may run at the same time. This can lead to a scenario where too many concurrent executions thrash the event loop, causing performance degradation and unresponsiveness. 
+
+To prevent this, you can set a concurrency limit for production executions in regular mode. This allows for more than one production execution to run concurrently, and queues up any concurrent production executions over the limit. These executions remain in the queue until concurrency capacity frees up, and are then processed in FIFO order.
```

Comment:
`This allows for more than one production execution to run concurrently` → Not necessarily, e.g. if you set `N8N_CONCURRENCY_PRODUCTION_LIMIT=1`. I went with `a number of executions` to keep it generic but maybe there's a better option.

Reply:
Have rephrased - let me know what you think.

Reply:
LGTM